src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -20,6 +20,39 @@
 
 namespace Stratis.Bitcoin.Features.Miner
 {
+    /// <summary>
+    /// PosMinting is used in order to generate new blocks. It involves a sort of lottery, similar to proof-of-work,
```

Comment:
always use `<see>` for everything that exists and you want to refer to it, i.e. all classes, constants etc. but not in formulas (like target formula below), but elsewhere, always use it, i won't repeat this comment, but it should be applied everywhere